src/components/dashboard/Dashboard.jsx

```diff
@@ -181,6 +181,13 @@ export default function Dashboard() {
     };
   }, [activeCourse, isLoading]);
 
+  const defCourseDetailValues = (item, key, callback) => {
+    if (!(key in item) || item[key] === null || item[key] === undefined) {
```

Comment:
```suggestion
    if (!(key in item) || !item[key]) {
```
This way we will test for falsy values like empty strings as well?